.circleci/cimodel/data/dimensions.py

```diff
@@ -9,6 +9,7 @@
 ROCM_VERSIONS = [
     "3.10",
     "4.0",
+    "4.0.1",
```

Comment:
Perhaps we can remove `4.0` if we're going to be adding `4.0.1`, or maybe instead we can just have the `4.0` image be `4.0.1`?

Reply:
Fixed.  Thanks for the suggestion.